ethcore/src/log_entry.rs

```diff
@@ -37,16 +39,25 @@ impl Encodable for LogEntry {
 	}
 }
 
-impl LogEntry {
-	/// Create a new log entry.
-	pub fn new(address: Address, topics: Vec<H256>, data: Bytes) -> LogEntry {
```

Comment:
I removed it, cause all `LogEntry` fields are public and I find explicit structure initialization more readable.